nextflow_schema.json

```diff
@@ -409,6 +409,17 @@
         "shortread_complexityfilter_fastp_threshold": {
             "type": "integer",
             "default": 30
+        },
+        "run_motus": {
+            "type": "boolean"
+        },
+        "download_motus_db": {
+            "type": "boolean"
+        },
+        "motus_downloaddb_script": {
+            "type": "string",
+            "default": "https://raw.githubusercontent.com/motu-tool/mOTUs/master/motus/downloadDB.py",
```

Comment:
I'm not a fan of always using latest master here. We should at least refer to a specific commit hash. Also, is there a reason the script could not be bundled with the MOTUS_DOWNLOADDB module?

Reply:
Not in `bin` but as within a `templates` directory, this is now supported.

Reply:
Oh interesting, I missed this. Any examples?

Reply:
The thing is that for different version of mOTUs, the downloadDB are different. By `bin templates`, the script will be fixed, right?

Reply:
Yes, putting the script in a templates directory, it will be fixed. But I actually think that's a good thing and, of course, the mOTUs version is also fixed. So we only have to ensure that the version in the module and the version of the script match.

Reply:
I removed the download script. So no need to worry about this part in current version.

Reply:
As the database downloading has been removed entirely, I think this point is moot now indeed :+1: